src/lib/backend.js

```diff
@@ -1,7 +1,5 @@
 // copied from https://github.com/aeternity/superhero-ui/blob/178bc2d63cb362ddc3b2163fed58deb2e253ec00/src/utils/backend.js
```

Comment:
We should remove this as its no longer the case this is matching the code at the provided URL.

Reply:
This is a reminder of a duplicate code that would be better to extract somehow later.